source/mainnet/smart-contracts/tutorials/piggy-bank/preparing.rst

```diff
@@ -0,0 +1,235 @@
+.. _Rust: https://www.rust-lang.org/
+
+.. _piggy-bank-preparing:
+
+====================================
+Preparing a testnet node on a server
+====================================
+
+This is the third :ref:`part of a tutorial<piggy-bank>` on smart contract
+development.
+So far you have written and tested a piggy bank smart contract in the Rust_ programming
+language.
+This part will focus on how you can set up your own testnet node on a server, and create a testnet account.
+In more detail, you will create an instance on a server that will host your own testnet node. Secondly, you will create your own testnet account and prepare it so you can use it in the fourth :ref:`part of this tutorial<piggy-bank-deploying>` for deploying and interacting with smart contracts.
+
+.. warning::
+
+   The reader is assumed to have basic knowledge of what a blockchain and smart
+   contract is, and some experience with Rust_.
+
+
+Preparation
+===========
+
+Before you start, make sure you create an account at your favourite cloud provider to set up your instance.
+
+You are now ready to select an instance on the cloud provider that matches the following requirements to run a testnet node:
+
+.. list-table::
+   :widths: 25 25
+   :header-rows: 1
+
+   * - Hardware (Testnet node)
+     - Recommended
+   * - CPU (Core)
+     - ?
+   * - RAM (Memory)
+     - 8 GB
+   * - Storage
+     - 50 GB
+   * - Operation system
+     - Ubuntu 20.04 x64
+
+.. Note::
+   The above requirements are simple and meant for a testnet node only. Please have a look at the requirements for mainnet nodes in the :ref:`run a node <node-requirements>` chapter.
+
+Syncing a testnet node
+======================
+
+The :ref:`run a node <node-requirements>` guide will help you set up your instance correctly. It is recommended using an ubuntu instance on a server in the cloud for this tutorial.
```

Comment:
The link 'node-requirements' is actually at the top of the 'run a node' page (not ideal naming but that link was already used before). Since I think that the confusion comes from the previous sentence that talks/links to the actual mainnet node requirements, I added a new link `system-requirements-node-mainnet` in the previous sentence and replaced it in the previous sentence.